lib/constants.rb

```diff
@@ -0,0 +1,23 @@
+module TicTacToe 
```

Comment:
Okay, let me work on figuring out how best to de-clutter `UI` and determining whether constants need to belong to a specific class. If they need to be accessed in several classes, I may leave them in the `Constants` file.